src/solana/solanaTypes.ts

```diff
@@ -151,5 +151,13 @@ export type SolanaInfoPayload = ReturnType<typeof asSolanaInfoPayload>
 
 export const asSolanaNetworkLocation = asObject({
   contractAddress: asString,
```

Comment:
Perhaps it's high-time to rename this to `ownerAddress` to follow the typical naming of things in solana world